src/domUpdates.js

```diff
@@ -5,36 +5,54 @@ const domUpdates = {
   },
 
   showCustomerDashboard() {
+    this.hideAll();
```

Comment:
actually googled hideAll() to see if it was a built in function. Great helper method/function